sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -2141,7 +2141,8 @@ abstract class DDLSuite extends QueryTest with DDLSuiteBase {
       root = Utils.createTempDir().getCanonicalPath, namePrefix = "addDirectory")
     val testFile = File.createTempFile("testFile", "1", directoryToAdd)
     spark.sql(s"ADD FILE $directoryToAdd")
-    assert(new File(SparkFiles.get(s"${directoryToAdd.getName}/${testFile.getName}")).exists())
+    val dirPath = spark.sql("LIST FILE").collect().map(_.getString(0)).head
```

Comment:
Can you try to construct the path instead? This feels a bit too much like a work around?

Reply:
I reverted this change to left it to future (when enabling isolation for Hive).